tools/tuhi-kete.py

```diff
@@ -332,8 +332,6 @@ class Worker(GObject.Object):
     def __init__(self, manager, args=None):
         GObject.GObject.__init__(self)
         self.manager = manager
-        self._run = self.run
-        self._stop = self.stop
```

Comment:
The reason for those were because python subclassing sometimes produce unexpected side effect.

Anyway, this doesn't matter and can be dropped indeed (though the Worker class will disappear, so really it doesn't matter)